### PR TITLE
feat(fuselage): new `Bubble` component

### PR DESCRIPTION
**Pull request by juliajforesti**
[DSAR-380](https://rocketchat.atlassian.net/browse/DSAR-380)

## Proposed changes (including videos or screenshots)

image

## Further comments

[DSAR-380]: https://rocketchat.atlassian.net/browse/DSAR-380?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/fuselage/src/components/Bubble/Bubble.styles.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: lengths.size(28);
+
+    padding-inline-start: lengths.padding(12);
+    padding-inline-end: lengths.padding(16);
+
+    border-radius: 20px;
```
Comment (dougfabris) [inline packages/fuselage/src/components/Bubble/Bubble.styles.scss:29]: if we will accept a new size shouldn't we review the `lenghts.border-raidius`?

File: packages/fuselage/src/components/Bubble/Bubble.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }`}
+    {...props}
+  >
+    <button className='rcx-bubble--content' onClick={onClick} {...contentProps}>
+      {icon && <Icon name={icon} size='x16' />}
+      <span>{children}</span>
+    </button>
+    {onDismiss && (
+      <button
+        aria-label={`Dismiss ${children}`}
```
Comment (dougfabris) [inline packages/fuselage/src/components/Bubble/Bubble.tsx:37]: shouldn't we be able to translate this dismiss as well?

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: packages/fuselage/src/components/Bubble/Bubble.tsx (hunk 2 above)
Comment (juliajforesti) [reply packages/fuselage/src/components/Bubble/Bubble.tsx:37]: yes indeed, thanks!

Comment (juliajforesti) [reply packages/fuselage/src/components/Bubble/Bubble.tsx:37]: You can overwrite the aria-label via the `dismissProps` and send the translated one

File: packages/fuselage/src/components/Bubble/Bubble.styles.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+@use '../../styles/variables/buttons.scss' as buttonColors;
+@use '../../styles/primitives/button.scss';
+@use '../../styles/mixins/interactivity.scss';
+
+.rcx-bubble {
+  display: flex;
+  align-items: center;
+
+  &--content,
```
Comment (dougfabris) [inline packages/fuselage/src/components/Bubble/Bubble.styles.scss:13]: let's try `_` for elements and leave `--` for modifiers

**Review by dougfabris — CHANGES_REQUESTED** [review]

**Review by dougfabris — APPROVED** [review]
LGTM!

**Pull request merged**